### ISSUE TITLE
forms: user sign-up fails due when profile form is enabled.

### ISSUE THREAD

**Issue by lnielsen** [inveniosoftware/invenio-userprofiles#30]
```
Traceback (most recent call last):
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/werkzeug/wsgi.py", line 659, in __call__
    return app(environ, start_response)
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/flask_debugtoolbar/__init__.py", line 125, in dispatch_request
    return view_func(**req.view_args)
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/flask_security/decorators.py", line 225, in wrapper
    return f(*args, **kwargs)
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/flask_security/views.py", line 117, in register
    user = register_user(**form.to_dict())
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/flask_security/registerable.py", line 29, in register_user
    user = _datastore.create_user(**kwargs)
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/flask_security/datastore.py", line 167, in create_user
    user = self.user_model(**kwargs)
  File "<string>", line 4, in __init__
    
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/sqlalchemy/orm/state.py", line 306, in _initialize_instance
    manager.dispatch.init_failure(self, args, kwargs)
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/sqlalchemy/util/langhelpers.py", line 60, in __exit__
    compat.reraise(exc_type, exc_value, exc_tb)
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/sqlalchemy/util/compat.py", line 183, in reraise
    raise value
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/sqlalchemy/orm/state.py", line 303, in _initialize_instance
    return manager.original_init(*mixed[1:], **kwargs)
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/sqlalchemy/ext/declarative/base.py", line 649, in _declarative_constructor
    setattr(self, k, kwargs[k])
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/sqlalchemy/orm/attributes.py", line 224, in __set__
    instance_dict(instance), value, None)
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/sqlalchemy/orm/attributes.py", line 807, in set
    value = self.fire_replace_event(state, dict_, value, old, initiator)
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/sqlalchemy/orm/attributes.py", line 829, in fire_replace_event
    self._replace_token or self._init_append_or_replace_token())
  File "/Users/lnielsen/envs/zenodo3/lib/python3.5/site-packages/sqlalchemy/orm/attributes.py", line 1156, in emit_backref_from_scalar_set_event
    child_state, child_dict = instance_state(child),\
AttributeError: 'dict' object has no attribute '_sa_instance_state'
```

**Comment by lnielsen** [comment]
``**form.to_dict()`` contains the ``profile`` key

**Comment by lnielsen** [comment]
Fixed in d55127181d7704d9fd4f7922bdc75ba7c468aa2d
